src/operations/liquidity_pool_withdraw.js

```diff
@@ -0,0 +1,49 @@
+import isUndefined from 'lodash/isUndefined';
+import xdr from '../generated/stellar-xdr_generated';
+
+/**
+ * Creates a liquidity pool withdraw operation.
+ *
+ * @function
+ * @alias Operation.liquidityPoolWithdraw
+ * @see https://developers.stellar.org/docs/start/list-of-operations/#liquidity-pool-withdraw
+ *
+ * @param {object} opts - Options object
+ * @param {string} opts.liquidityPoolId - The liquidity pool ID.
+ * @param {string} opts.amount - Amount of pool shares to withdraw.
+ * @param {string} opts.minAmountA - Minimum amount of first asset to withdraw.
+ * @param {string} opts.minAmountB - Minimum amount of second asset to withdraw.
+ * @param {string} [opts.source] - The source account for the operation. Defaults to the transaction's source account.
+ *
+ * @returns {xdr.Operation}   The resulting operation (xdr.LiquidityPoolWithdrawOp).
+ */
+export function liquidityPoolWithdraw(opts = {}) {
+  const attributes = {};
+  if (!opts.liquidityPoolId) {
+    throw new TypeError('liquidityPoolId argument is required');
+  }
+  attributes.liquidityPoolId = xdr.PoolId.fromXDR(opts.liquidityPoolId, 'hex');
+
+  if (isUndefined(opts.amount)) {
```

Comment:
Same here as above: `this.isValidAmount(opts.amount)` is the preferred way to validate asset quantities.

Reply:
Cool! changed to `Operation.isValidAmount` in https://github.com/stellar/js-stellar-base/pull/454/commits/43de4f1ed5d0abd37faa3b28401d8f5b632e52a3.